tuftool/src/root.rs

```diff
@@ -94,7 +94,7 @@ pub(crate) enum Command {
         /// Path to root.json
         path: PathBuf,
         #[structopt(parse(try_from_str = parse_key_source))]
-        key_source: Box<dyn KeySource>,
+        key_source: Vec<Box<dyn KeySource>>,
```

Comment:
If we're going to accept multiple `KeySource`s, it probably makes sense to add a short flag to this argument. (Generally good CLI practice as well)

So when you call it it would look like:
`tuftool .... -k keysource1 -k keysource2`

I believe you need to add `short = "k"` to the `structopt` annotations.